src/main/java/sleepy/tasks/ToDo.java

```diff
@@ -0,0 +1,29 @@
+package sleepy.tasks;
+
+/**
+ * This class is a type of task.
+ *
+ * @author kjw142857
+ */
+public class ToDo extends Task {
```

Comment:
Would renaming to ToDoTask be better? Similar reason to the previous comments but in this case I felt like ToDo is more of a collection class (although without s) than a task.